docs/guides/integration/alternative-indexes.md

```diff
@@ -62,6 +62,58 @@ $ # Configure the index URL with the username
 $ export UV_EXTRA_INDEX_URL=https://VssSessionToken@pkgs.dev.azure.com/{organisation}/{project}/_packaging/{feedName}/pypi/simple/
 ```
 
+## Google Artifact Registry
+
+uv can install packages from
+[Google Artifact Registry](https://cloud.google.com/artifact-registry/docs). Authenticate to a
+repository using password authentication or using [`keyring`](https://github.com/jaraco/keyring)
```

Comment:
What about using a Service Account?

That's how I approach it in a CI environment, at least. Following this [Google guide](https://cloud.google.com/docs/authentication#auth-decision-tree) too.
Not sure if you have a better approach.

Reply:
I can quickly mention other authentication methods indeed :+1: